ui/src/app/cohort-review/detail-page/detail-page.ts

```diff
@@ -12,19 +15,55 @@ export class DetailPage implements OnInit, OnDestroy {
 
   sidebarOpen = true;
   participant: Participant;
-  subscription: Subscription;
-  participantId: number;
-  constructor(private route: ActivatedRoute) {}
+  annotations: ParticipantCohortAnnotation[];
+  annotationDefinitions: CohortAnnotationDefinition[];
+  subscriptions: Subscription[] = [];
+  constructor(
+    private reviewAPI: CohortReviewService,
```

Comment:
I would name these `reviewService` and `defsService` because we have a distinction between service and API. (Service is generated for angular, and API is generated for fetch)

Reply:
I agree with the reasoning, but there is an established pattern of calling these 'API' in the angular code, so I'm inclined to keep that consistency to make the refactoring smoother.